src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -204,6 +205,7 @@ const ArticlePage = ({ pageData }: { pageData: Article }) => {
       topStoriesContent ? (
         <ExperimentTopStories topStoriesContent={topStoriesContent} />
       ) : null,
+    jumpTo: JumpTo,
```

Comment:
Could/should we add a Storybook example for an article with a JumpTo component?

Reply:
I've amended an existing News article fixture to include JumpTo and added a story using that data 👍